database/migrations/2024_05_03_122855_create_reservations_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('reservations', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('user_id');
+            $table->unsignedBigInteger('area_id');
+            $table->date('date');
+            $table->integer('fee_log');
```

Comment:
fee_log is a column to record the fee unit price at the time of reservation.
If the admin changes the unit price on the fee table, when the user views the reservation history, it will change to the amount at the time of reservation, so the unit price is recorded in the reservation table at the time of reservation. .

Reply:
@Tomo-Furu can you also use double for this?

Reply:
@kredo-school    
Yes, I changed it to double.
`            $table->double('fee_log', 5, 2);`